.gitignore

```diff
@@ -18,4 +18,7 @@
 *.sublime-workspace
 
 # Custom
-coverage.txt
\ No newline at end of file
+coverage.txt
+
+# tmp directory was generated by example_remoteVerify_test.go 
```

Comment:
I'm not able to find `example_remoteVerify_test.go` in this PR.

Reply:
https://github.com/notaryproject/notation-go/blob/5e98995bd17a4ed9ad435c7e6d240904d183ed56/example_remoteVerify_test.go#L127-L131

Every time the unit test runs locally, it generates a temporary file, which is not convenient.